pkg/scheduler/objects/allocation_ask.go

```diff
@@ -262,16 +261,9 @@ func (aa *AllocationAsk) HasTriggeredPreemption() bool {
 	return aa.preemptionTriggered
 }
 
-// SetLastPreemptionAttempt sets the time which tells when the last preemption attempt was performed for this ask.
-func (aa *AllocationAsk) SetLastPreemptionAttempt(attempt time.Time) {
+// test only
```

Comment:
This is only called from the objects pkg, so can be moved to a *_test.go file. No need to pollute the main package with test code.

Reply:
Done